sdk/selfhelp/arm-selfhelp/README.md

```diff
@@ -53,14 +53,15 @@ const { HelpRP } = require("@azure/arm-selfhelp");
 const { DefaultAzureCredential } = require("@azure/identity");
 // For client-side applications running in the browser, use InteractiveBrowserCredential instead of DefaultAzureCredential. See https://aka.ms/azsdk/js/identity/examples for more details.
 
-const client = new HelpRP(new DefaultAzureCredential());
+const subscriptionId = "00000000-0000-0000-0000-000000000000";
+const client = new HelpRP(new DefaultAzureCredential(), subscriptionId);
```

Comment:
I think we should revert `subscriptionId` change. Then create an issue to report the readme issue.